judgments/urls.py

```diff
@@ -47,6 +48,12 @@
         PdfDetailView.as_view(),
         name="weasy_pdf",
     ),
+    re_path(
+        r"^/*(?P<original_url>.*/press-summary)/?$",
+        lambda request, original_url: HttpResponsePermanentRedirect(
+            f"/{original_url}/1"
```

Comment:
I think this is fine but I don't think I can prove it.

Reply:
I think this would cause something like `caselaw.nationalarchives.gov.uk/malicious-site.com/press-summary` to try redirect to `/malicious-site.com`, which I think some browsers may interpret as an incorrectly typed URL and try to silently catch when we serve a 404.